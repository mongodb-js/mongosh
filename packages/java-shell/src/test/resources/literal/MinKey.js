// command checkResultClass
new MinKey();
// command checkResultClass
MinKey()