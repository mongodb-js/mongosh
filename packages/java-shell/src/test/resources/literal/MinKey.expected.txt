MinKeyResult: { "$minKey" : 1}
MinKeyResult: { "$minKey" : 1}